airflow-core/src/airflow/models/hitl.py

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import sqlalchemy_jsonfield
+from sqlalchemy import Boolean, Column, String, Text
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.ext.hybrid import hybrid_property
+
+from airflow.models.base import Base
+from airflow.settings import json
+from airflow.utils.sqlalchemy import UtcDateTime
+
+
+class HITLResponseModel(Base):
+    """Human-in-the-loop received response."""
+
+    __tablename__ = "hitl_response"
+    ti_id = Column(
+        String(36).with_variant(postgresql.UUID(as_uuid=False), "postgresql"),
+        primary_key=True,
+        nullable=False,
+    )
+
+    # Input Request
+    options = Column(sqlalchemy_jsonfield.JSONField(json=json), nullable=False)
+    subject = Column(Text, nullable=False)
+    body = Column(Text, nullable=True)
+    default = Column(sqlalchemy_jsonfield.JSONField(json=json), nullable=True)
+    multiple = Column(Boolean, unique=False, default=False)
+    params = Column(sqlalchemy_jsonfield.JSONField(json=json), nullable=False, default={})
+
+    # Response Content Detail
+    response_at = Column(UtcDateTime, nullable=True)
+    user_id = Column(String(128), nullable=True)
+    response_content = Column(
+        sqlalchemy_jsonfield.JSONField(json=json),
+        nullable=True,
+        default=None,
+    )
+    params_input = Column(sqlalchemy_jsonfield.JSONField(json=json), nullable=False, default={})
```

Comment:
As comment above: woult propose to have this nullable as well to signal no input (yet) like the three fields above.
```suggestion
    params_input = Column(sqlalchemy_jsonfield.JSONField(json=json), nullable=True)
```

Reply:
let's keep this discussion in https://github.com/apache/airflow/pull/52868#discussion_r2193880430